layouts/partials/post_meta/spec_meta.html

```diff
@@ -41,7 +41,22 @@
             </td>
           {{- end -}}
         </tr>
-      {{- end -}}
+      {{- $N := len (or $val "") }} 
+      {{- if (and (eq $attr "endorsed-by") (le $N 1)) }}
+          <tr class="field">
```

Comment:
```suggestion
          {{ /* add blank line to table */ }}
          <tr class="field">
```

Reply:
Hugo complained about the spaces (i.e., `{{ /*` and `*/ }}`).  But it worked without the spaces (i.e., `{{/*` and `*/}}`).